README.rst

```diff
@@ -21,89 +21,131 @@ PennyLane Cirq Plugin
     :alt: PyPI
     :target: https://pypi.org/project/pennylane-cirq
 
+.. image:: doc/_static/puzzle_cirq.png
+    :align: center
+    :width: 200px
+    :target: javascript:void(0);
```

Comment:
Wondering about this one, is there any reason to have it? (Didn't spot it with the Qiskit one but now I see it's there as well)

Reply:
Seems to create the feeling that one could be redirected to another page

Reply:
By default, Sphinx adds a hyperlink to all images, so that clicking on the image takes you to the source file. This is a (hacky) way of turning it off. Back when we first wrote the SF documentation, we got internal feedback that users didn't like how clicking the image navigated you away from the page.